x-pack/solutions/observability/plugins/streams/server/routes/streams/processing/route.ts

```diff
@@ -35,7 +32,7 @@ const paramsSchema = z.object({
   }),
 });
 
-type ProcessingSimulateParams = z.infer<typeof paramsSchema>;
+export type ProcessingSimulateParams = z.infer<typeof paramsSchema>;
```

Comment:
not a biggie, but would like us to get into the habit of using `= z.ZodSchema<ExplicitType>`

Reply:
also should we move this into the schema package then?

Reply:
+1 for z.Schema, but I think we don't need to move yet because it's only used internally in the server, not from the client.